packages/backend-function/src/factory.ts

```diff
@@ -233,12 +235,23 @@ class AmplifyFunction
 
     const shims =
       runtime === Runtime.NODEJS_16_X
-        ? // this shim includes the cjs shim because it's required for the v2 sdk to work
-          [require.resolve('./lambda-shims/resolve_ssm_params_sdk_v2_shim')]
-        : [
-            require.resolve('./lambda-shims/cjs_shim'),
-            require.resolve('./lambda-shims/resolve_ssm_params_shim'),
-          ];
+        ? []
+        : [require.resolve('./lambda-shims/cjs_shim')];
+
+    const ssmResolverFile =
+      runtime === Runtime.NODEJS_16_X
+        ? require.resolve('./lambda-shims/resolve_ssm_params_sdk_v2') // use aws cdk v2 in node 16
+        : require.resolve('./lambda-shims/resolve_ssm_params');
+
+    const invokeSsmResolverFile = require.resolve(
+      './lambda-shims/invoke_ssm_shim'
+    );
+
+    const bannerCode = readFileSync(ssmResolverFile, 'utf-8')
```

Comment:
I think since we are adding the banner again we should reintroduce a comment saying what this single line of code is doing.